core/src/main/java/org/infinispan/configuration/cache/L1ConfigurationBuilder.java

```diff
@@ -105,7 +123,7 @@ public void validate() {
          if (!clustering().cacheMode().isDistributed())
             throw CONFIG.l1OnlyForDistributedCache(clustering().cacheMode().friendlyCacheModeString());
 
-         if (attributes.attribute(LIFESPAN).get() < 1)
+         if (attributes.attribute(LIFESPAN).get().longValue() < 1)
```

Comment:
The value of `1` here was minutes before? Or even 1ms was valid? Otherwise, we'll need to update here since the `.longValue()` will return the time in ms.

Reply:
It's always been milliseconds. Introducing a minimum threshold may be a good idea, but not something we should do here.